canopen_monitor/parser/pdo.py

```diff
@@ -58,7 +67,7 @@ def parse_pdo(num_elements, pdo_type, eds, data):
     output_string = ""
     data_start = 0
     for i in range(num_elements, 0, -1):
-        pdo_definition = int(eds[pdo_type].sub_indices[i].default_value).to_bytes(4, "big")
+        pdo_definition = int(eds[pdo_type].sub_indices[i].default_value, 16).to_bytes(4, "big")
```

Comment:
Include check for base16 (see code in slack) also we can remove sub_indices

Reply:
Also, wrap in try block to catch NoneValueError in case index or sub-index is invalid